desktop/package/package.gradle

```diff
@@ -328,12 +343,14 @@ task packageInstallers {
             jpackageTempDir.deleteDir()
             jpackageTempDir.mkdirs()
 
-            executeCmd(jPackageFilePath + commonOpts +
-                    " --dest \"${jpackageTempDir}\"" +
-                    " --type app-image")
+            // Copy over app-image for AppImage packaging
+            copy {
+                from jpackageAppImage
+                into new File(jpackageTempDir, "Haveno")
```

Comment:
How is this hardcoded string "Haveno" used?

Just ensuring it's only temporary in the build process, since the application name / data directory can be customized for different networks, e.g. "Haveno-example" in other places like build.yml and HavenoExecutable.java

Reply:
By default, the AppImage building process expects a directory Haveno as a result of making an app-image directory so this is required to prevent changing the AppImage building process.